docs/getting-started/edfi-exchange/readme.md

```diff
@@ -0,0 +1,46 @@
+# Ed-Fi Exchange
+
+The Ed-Fi Exchange is a technology hub for community contributions aligned to the Ed-Fi Data Standard and Implementation Suite.
+
+Find out how it works and how to contribute.
+
+## Browse the Exchange
+
+* [Code & Technology](./code-technology/readme.md)
+* [Guides & Documentation](./guides-documentation/readme.md)
+* [How it Works](./how-it-works.md)
+* [How to Contribute](./how-to-contribute.md)
+
+## Cloud Deployment  Strategies
+
+One of the most common types of questions the Ed-Fi Alliance receives from new community members is "how do I install and run this software?" Increasingly, that question is posed with respect to cloud deployments. Given that the Alliance does not run its own software, and given the myriad ways of optimizing a deployment, it is difficult for us to have the expertise and bandwidth to provide detailed guidance on deployment architecture and security. At the same time, we are in frequent conversation with community members and can learn from them and reflect some practices back to the community. In that light we offer the following documents and tools that may be helpful in planning your own deployments. Some have been developed at the request of the Alliance, and other solutions are offered by the community. Please recognize the necessary limitations and "use at your own risk."
+
+### Amazon Web Services
+
+Education Analytics StartingBlocks - StartingBlocks is a production ready deployment of the Ed-Fi ODS/API in Amazon Web Services (AWS)
+
+### Azure
+
+Deploying the Ed-Fi ODS API 6x on Azure - using App Services
+
+ODS/API 5.3 Deployment on Azure from K12 Analytics Engineering - using App Services
+
+### Google Cloud Platform
+
+Ed-Fi Kubernetes Deployment - 💡 also a good starting point for Kubernetes on other platforms)
+
+How To: Run Ed-Fi on Google Cloud - using Cloud Run
```

Comment:
Should we remove these sections above for now, before we publish the initial framework?

I like the idea of getting a small pull request completed soon, so that it will be easy for multiple contributors to submit documents into the new folder structure.

Reply:
Ah I didn't even realize I moved these over without links. Yes I am removing them.